js/private/js_image_layer.bzl

```diff
@@ -490,6 +490,9 @@ js_image_layer_lib = struct(
         "platform": attr.label(
             doc = "Platform to transition.",
         ),
+        "generate_empty_layers": attr.bool(
+            doc = "Generate layers even if they are empty.",
```

Comment:
should it say "only intended for rules-docker" and/or "deprecated, to be removed in 3.0" or sth?

Reply:
its a simple enough option that it can stick around;; but I will mention what it is intended for